### PR TITLE
Add 8.1 branch to docs build

### PR DESCRIPTION
**Pull request by jrodewig**
* Creates a new `/stack/versions/8.1.asciidoc` file copied from `master.asciidoc`
* Updates `conf.yaml` to include the 8.1 branch

### CODE REVIEW

File: shared/versions/stack/8.1.asciidoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
+:version:                8.1.0
+////
+bare_version never includes -alpha or -beta
+////
+:bare_version:           8.1.0
+:logstash_version:       8.1.0
+:elasticsearch_version:  8.1.0
+:kibana_version:         8.1.0
+:apm_server_version:     8.1.0
+:branch:                 master
```
Comment (lcawl) [inline shared/versions/stack/8.1.asciidoc:10]: I think this needs to be 8.1 instead:
```suggestion
:branch:                 8.1
```

Comment (jrodewig) [reply shared/versions/stack/8.1.asciidoc]: Good catch! Thanks!

**Comment by jrodewig** [comment]
@bmorelli25 It looks like the https://github.com/elastic/apm-aws-lambda repo hasn't cut an 8.1 branch yet. Are we expecting an 8.1 branch or should I update the conf to exclude it?

Feel free to push commits directly to this branch if that's easier.

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs rebuild

**Comment by jrodewig** [comment]
It looks like there a few broken links related to the ES Rust client and the Enterprise Search Python client. I'll reach out to the clients team and see when we can expect 8.1 branches for those docs.

**Comment by jrodewig** [comment]
@elasticmachine run elasticsearch-ci/docs rebuild

**Review by lcawl — APPROVED** [review]
LGTM, thanks!

**Comment by jrodewig** [comment]
Thanks, Lisa!

**Pull request merged**